types/protractor-helpers/index.d.ts

```diff
@@ -44,7 +44,7 @@ declare namespace protractor {
 //        jasmine d.ts.
 
 declare namespace jasmine {
-  interface Matchers {
+  interface Matchers<T> {
```

Comment:
request from @paulvanbrenk (see above) so that travis would build the original file changes successfully.